app/models/patient.rb

```diff
@@ -122,15 +122,54 @@ def latest_transfer
 
   # Patients who are eligible for reminders
   scope :reminder_eligible, lambda {
```

Comment:
This is going to need a method comment describing the logic here, as it's a huge scope that is not super readable.